drivers/unifiedPlatform/stworkflows/pds/workflowDataservice.go

```diff
@@ -149,7 +159,12 @@ func (wfDataService *WorkflowDataService) GetDsDeploymentResources(deployment ma
 	deploymentName, deploymentId := GetDeploymentNameAndId(deployment)
 	log.Debugf("deployment Name [%s] and Id [%s]", deploymentName, deploymentId)
 
-	dbConfig, err = dslibs.GetDeploymentConfigurations(namespace, dataServiceName, deploymentName)
+	podName, err := wfDataService.GetDeploymentPodName(deploymentId)
```

Comment:
can we directly call `_, podName, err := dslibs.GetDeployment(deploymentId)` instead of having a seperate workflow only to get `GetDeploymentPodName`